### PR TITLE
when the user pass --walk should not consider ignorable files

### PR DESCRIPTION
**Pull request by guilherme**
I don't know if it's the best approach/code but it fixes when the user passes --all

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -64,10 +64,12 @@ fn main() {
         if file_type.is_file() && config.show_files { println!("{}", path); }
     };
 
-
-    let ignorablable_filesources = vec![".gitignore", ".agignore"];
-
+    let mut ignorablable_filesources = Vec::new();
```
Comment (sebglazebrook) [inline src/main.rs:67]: it would be more rusty to write

```
let mut ignorablable_filesources = vec![];
```

**Comment by guilherme** [comment]
After the fix:
➜  walk git:(master) target/debug/walk --all ~/Development/ | wc -l
  362821
➜  walk git:(master) find ~/Development | wc -l
  363014

closer.. I still have to investigate what are these files that are missing

Before the fix:
➜  walk git:(master) time walk --all ~/Development/ | wc -l
  287144

**Comment by guilherme** [comment]
curiosity:
➜  walk git:(master) time target/debug/walk --all ~/Development/ | wc -l
  362821
target/debug/walk --all ~/Development/  1,31s user 1,58s system 78% cpu 3,671 total
wc -l  0,09s user 0,64s system 19% cpu 3,670 total

➜  walk git:(master) time find ~/Development/ | wc -l
  363014
find ~/Development/  0,45s user 1,93s system 49% cpu 4,855 total
wc -l  0,06s user 0,02s system 1% cpu 4,854 total

with ignore (before fix):
➜  walk git:(master) time walk --all ~/Development/ | wc -l
  287144
walk --all ~/Development/  40,83s user 2,39s system 94% cpu 45,537 total
wc -l  0,12s user 1,11s system 2% cpu 45,536 total

File: src/main.rs [hunk 2]
```diff
@@ -64,10 +64,12 @@ fn main() {
...
     let mut ignorer = Ignorer::new();
+    if !config.show_ignorable {
```
Comment (sebglazebrook) [inline src/main.rs:69]: Maybe use #append here?

https://doc.rust-lang.org/std/vec/struct.Vec.html#method.append

File: src/main.rs (hunk 1 above)
Comment (guilherme) [reply src/main.rs:67]: 👍 thanks!!

File: src/main.rs (hunk 2 above)
Comment (guilherme) [reply src/main.rs:69]: way better! thanks!

**Pull request closed without merging**